src/cljs/org/broadinstitute/firecloud_ui/page/profile.cljs

```diff
@@ -178,9 +178,13 @@
 (react/defc Page
   {:render
    (fn [{:keys [this props state]}]
-     (let [new? (:new-registration? props)]
+     (let [new? (:new-registration? props)
+           update? (:update-registration? props)]
        [:div {:style {:marginTop "2em"}}
-        [:h2 {} (if new? "New User Registration" "Profile")]
+        [:h2 {} (cond
```

Comment:
It does the right thing for the update case. If new, uses "Register", otherwise uses "Save Profile".